cmd/spicedb/servetesting_integration_test.go

```diff
@@ -39,10 +40,18 @@ func TestTestServer(t *testing.T) {
 	require.NoError(err)
 	defer conn.Close()
 
+	resp, err := healthpb.NewHealthClient(conn).Check(context.Background(), &healthpb.HealthCheckRequest{Service: "authzed.api.v1alpha1.SchemaService"})
```

Comment:
@ecordell I mentioned it in the commit message: this won't flake because there is an "Eventually" already in the setup of the container, so it's guaranteed to have started? Or are you think about the container, even ready, taking too long to respond?